draft-enghardt-panrg-path-properties.md

```diff
@@ -46,8 +48,21 @@ This information is expressed as properties of paths between two endpoints.
 # Introduction
 
 Because the current Internet provides an IP-based best-effort bit pipe, endpoints have little information about paths to other endpoints.
-A Path Aware Network exposes information about one or multiple paths through the network to endpoints,
-so that endpoints can use this information.
+A Path Aware Network exposes information about one or multiple paths through the network to endpoints or the network infrastructure.
+
+It is impossible to provide an exhaustive list of path properties, as with every new technology and protocol, novel properties might become relevant.
+In order to specify broadly relevant path properties, scenarios in which knowledge about (additional) path information gives advantages are described here.
+
+Traffic policies leveraging path awareness can be defined.
+Such policies can allow or disallow sending traffic over specific networks or nodes, select an appropriate protocol depending on the capabilities of the on-path devices (e.g., using QUIC instead of TCP if no device blocks UDP traffic), or adjust protocol parameters to the used path.
+
+Network monitoring systems in a path aware network can associate their measurements with the actual path used, and observe if the measurements conform with the advertised path properties.
+Network operators can use the path information to specify comprehensive Quality of Service (QoS) guarantees.
```

Comment:
Here I'm also not quite sure I understand the use case(s).
What does it mean to "associate their measurements with the actual path used"? A monitoring system gets information from on-path devices, e.g., latency measurements, and then it can compare them to... what exactly? Path properties advertised where? In some contract that specifies certain latency properties? Or in some future path-awareness protocol that communicates some network performance to endpoints, and a different party (who?) wants to find out if this information is true?
In the second sentence, do you mean that a network operator could use on-path measurements to, e.g., write somewhere that their 95th quantile latency is x milliseconds, or that their median packet loss is x %? I'm not sure in what specific scenario this would actually happen.

Maybe it's easier to start with something simpler, where we only observe performance and don't promise or advertise anything yet. For example, "Network operators can use path properties, e.g., measured by on-path devices, to check whether recent end-user traffic satisfy specific Quality of Service (QoS) requirements, and identify potential problems with their network early on, before the end-user complains"?
Perhaps here we could cite something later on, like, IPPM or IPFIX or something? I'm not sure I fully understand the context in which these protocols are actually used though...

Reply:
I agree that it is probably better to make a simple statement that does not require many assumptions. I adjusted the example in the pull request. I think it is fine the way it is now since we only need these use cases to justify our choice of path properties and since most path properties can be relevant for QoS, we should be fine ;)

I'm also not an expert on either of these protocols, but I will take a look at them to see how to justify our properties.

Reply:
Cool, thanks!